apis/core.oam.dev/common/types.go

```diff
@@ -573,3 +573,29 @@ type ReferredObjectList struct {
 	// +optional
 	Objects []ReferredObject `json:"objects,omitempty"`
 }
+
+// ContainerState defines the state of a container
+type ContainerState string
+
+const (
+	// ContainerRunning indicates the container is running
+	ContainerRunning ContainerState = "Running"
+	// ContainerWaiting indicates the container is waiting
+	ContainerWaiting ContainerState = "Waiting"
+	// ContainerTerminated indicates the container is terminated
+	ContainerTerminated ContainerState = "Terminated"
+)
+
+// ContainerStateToString convert the container state to string
+func ContainerStateToString(state corev1.ContainerState) string {
+	switch {
+	case state.Running != nil:
+		return "running"
```

Comment:
Why small letter "running"  is returned? since capital letter "Running" is defined  in  `ContainerRunning ContainerState = "Running"`

Reply:
Sorry, this is my carelessness, I will fix it.

Reply:
I have fixed it.